literature.bib

```diff
@@ -1,3 +1,17 @@
+
+@inproceedings{PAM+:ICPE20,
+  title={Sampling Effect on Performance Prediction of Configurable Systems: A Case Study},
+  author={Juliana Alves Pereira and Mathieu Acher and Hugo Martib and Jean-Marc J{\'e}z{\'e}quel},
+  booktitle={Proceedings of the ACM/SPEC International Conference on Performance Engineering},
+  pages={277--288},
+  doi={https://doi.org/10.1145/3358960.3379137},
+  sampling-tags = {name=Distance-Based Sampling, feature model, population-based search, semi-automatic search, non-functional properties, evaluation, effectiveness, available tool, open-source tool, compares Coverage-based/Solver-based/Randomized solver-based/Distance-based/Diversified distance-based/Random},
```

Comment:
is distance-based really an algorithm proposed in this paper? is it the only one?

if it is only evaluated, remove tags on the algorithm

if there are multiple proposed, that use several entries (cf. sampling2-tags)